### DISCUSSION TITLE
Can the translation team be proactively notified of changes in translation fields?

### DISCUSSION THREAD

**Discussion by ZhangChengLin** [PHPAuth/PHPAuth#510] [General]
Set up a GitHub translation team and invite translators to enter. After updating some sentences, you do not need to notify specific people, you can directly @翻译 team to actively notify the changes.
This can increase the chance of synchronizing the local language as soon as possible.

**Comment by KarelWintersky** [comment]
Dunno how to set up it :(

**Comment by ZhangChengLin** [comment]
> Dunno how to set up it :(

In the GitHub organization, create a new group, then set up members, and then when the related pr merges, @ that group's name.

**Comment by Conver** [comment]
That would require us to actually have a translator team, which we don't.. We barely have people working on the code itself.

**Reply by ZhangChengLin** [reply]
Yes, if the code is still maintained, you can try it. Maybe someone is willing to contribute language translation.
I thought 2.0 would be there, but I didn't see it.
